src/OMSimulatorLib/Flags.h

```diff
@@ -143,6 +145,7 @@ namespace oms
       {"--logFile", "-l", "Specifies the logfile (stdout is used if no log file is specified)", re_default, Flags::LogFile, false},
       {"--logLevel", "", "0 default, 1 debug, 2 debug+trace", re_number, Flags::LogLevel, false},
       {"--maxEventIteration", "", "Specifies the max. number of iterations for handling a single event", re_number, Flags::MaxEventIteration, false},
+      {"--maxLoopIteration", "", "Specifies the max. number of iterations for solving algebraic loops", re_number, Flags::MaxLoopIteration, false},
```

Comment:
Could be clarified a bit, since it targets only system-level loops between components and doesn't affect the alg. loops inside components.

Reply:
Done